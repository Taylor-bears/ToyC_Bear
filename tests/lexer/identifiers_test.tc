int var1 = 1;
int _var2 = 2; // 如果你支持下划线开头的标识符
int v3ar_ = 3;
return 123var;