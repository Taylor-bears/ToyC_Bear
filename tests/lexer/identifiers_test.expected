INT
ID(var1)
ASSIGN
NUMBER(1)
SEMICOLON
INT
ID(_var2)
ASSIGN
NUMBER(2)
SEMICOLON
INT
ID(v3ar_)
ASSIGN
NUMBER(3)
SEMICOLON
RETURN
NUMBER(123)
ID(var)
SEMICOLON
EOF